framework/BaseYii.php

```diff
@@ -451,7 +469,7 @@ public static function info($message, $category = 'application')
      */
     public static function beginProfile($token, $category = 'application')
```

Comment:
Profiling doesn't belong to logging. Ideally it should be moved into separate component.